showcases/shared/textarea.json

```
@@ -188,6 +188,27 @@
 			}
 		]
 	},
+	{
+		"name": "Show Required Asterisk",
```

```suggestion
		"name": "Show asterisk for required form fields (textarea)",
```

I'm struggling whether it might be easier to understand when explaining it with some more words.